src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -958,19 +958,6 @@ public function getModel()
     }
 
     /**
-     * Get a fresh instance of a model instance being queried.
-     *
-     * @param  array  $attributes
-     * @return \Illuminate\Database\Eloquent\Model
-     */
-    public function newModel(array $attributes = [])
```

Comment:
Ah sorry... Well then we should really merge this quickly because private constructors did work before and they don't work now.